README.md

```diff
@@ -182,6 +182,69 @@ transformer.generate("wrappers").catch(err => {
+
+The `processReflect` processor hook has the signature `(idl, implName) => ({ get, set })`, where:
+
+- `idl` is the [`attribute` AST node](https://github.com/w3c/webidl2.js/#attribute-member) as emitted by the [webidl2](https://github.com/w3c/webidl2.js) parser.
+
+- `implName` (string) is a JavaScript expression that would evaluate to the implementation object at runtime.
+
+- `get` (string) is the code that will be output in the attribute getter, after any function prologue.
+
+- `set` (string) is the code that will be output in the attribute setter, after any function prologue.
+
+The hook also has a utility method that is accessible via `this`:
+
+- `addImport(path, [importedIdentifier])` utility to require external modules from the generated interface. This method accepts 2 parameters: `path` the relative or absolute path from the generated interface file, and an optional `importedIdentifier` the identifier to import. This method returns the local identifier from the imported path.
+
+The following variables are available in the scope of the replacement code:
+
+- `globalObject` (object) is the global object associated with the interface
+
+- `interfaceName` (string) is the name of the interface
+
+- (for setter only) `V` (any) is the converted input to the setter method.
+
+To mark an attribute as reflected, an extended attribute whose name starts with `Reflect` should be added to the IDL attribute. This means that any of the following is treated as reflected by webidl2js:
+
+```webidl
+[Reflect] attribute boolean reflectedBoolean;
+[ReflectURL] attribute USVString reflectedURL;
+[Reflect=value, ReflectURL] attribute USVString reflectedValue;
+```
+
+webidl2js itself does not particularly care about the particular reflection-related extended attribute(s) being used, only that one exists. However, your processor hook can make use of the extended attributes for additional information on how the attribute is reflected.
+
+An example processor function that implements `boolean` IDL attribute reflection is as follows:
+
+```js
+function processReflect(idl, implName) {
+  // Determine the name of the content attribute being reflected as follows:
+  // - if [Reflect] is provided with a value, then the value is the content attribute.
+  // - otherwise, the content attribute is the name of the IDL attribute, lowercased.
+  const reflectAttr = idl.extAttrs.find(attr => attr.name === "Reflect");
+  const attrName =
+    reflectAttr && reflectAttr.rhs && reflectAttr.rhs.value.replace(/_/g, "-") || idl.name.toLowerCase();
```

Comment:
Given that we (or at least I) want to move to strings anyway, we should probably either change or omit this part of the example?